ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -164,7 +164,7 @@ ANALYTICS_IDS = {
 
 AXES_LOCK_OUT_AT_FAILURE = False
 
-LUCENE_ENABLED = True
+LUCENE_ENABLED = False
```

Comment:
@snopoke Going through this cleanup branch I thought that Lucene was no longer used, and no one else thought it was used either. Sounds like definitely re-enable.